src/Workspaces/Core/Portable/Rename/IRemoteRenamerService.cs

```diff
@@ -204,7 +183,7 @@ internal sealed class SerializableRenameLocations
         // We use arrays so we can represent default immutable arrays.
```

Comment:
>// We use arrays so we can represent default immutable arrays [](http://example.com/codeflow?start=8&length=61)

This is odd. Perhaps it was a workaround for old version of JSON serializer that had a bug?